src/lib/loaders/loaders_with_authentication/exchange.ts

```diff
@@ -1,7 +1,15 @@
 import factories from "../api"
 import config from "config"
+import fetch from "node-fetch"
+import urljoin from "url-join"
+import { GraphQLError } from "graphql"
+import { verbose } from "lib/loggers"
 
-const { EXCHANGE_APP_ID } = config
+const { EXCHANGE_APP_ID, EXCHANGE_API_BASE } = config
+interface GraphQLArgs {
```

Comment:
I'm gonna be a bit nit-picky so please forgive 🙏  Line break after the const pls